src/reanimated2/shareables.ts

```diff
@@ -149,6 +149,12 @@ export function makeShareableCloneRecursive<T>(
               value.__stackDetails
             );
             delete value.__stackDetails;
+          } else if (value.__stackDetails) {
```

Comment:
I think we decided to use `__version` to determine that we have a dev bundle indeed but after some consideration `__stackDetails` indeed seems to be a better choice 👍